scri/SpEC/__init__.py

```diff
@@ -6,3 +6,5 @@
 
 from .com_motion import com_motion, estimate_avg_com_motion, remove_avg_com_motion
 from .file_io import read_from_h5, write_to_h5
+
+from .memory import mem_mass_aspect, mem_energy_flux, mem_angular_momentum_aspect, mem_angular_momentum_flux
```

Comment:
It seems like the main two functions in your `memory.py` script is `add_memory` and `BMS_strain`, did you mean to add those to this list?